crypto/cmp/cmp_client.c

```diff
@@ -106,9 +106,12 @@ static int save_statusInfo(OSSL_CMP_CTX *ctx, OSSL_CMP_PKISI *si)
     ss = si->statusString; /* may be NULL */
     for (i = 0; i < sk_ASN1_UTF8STRING_num(ss); i++) {
         ASN1_UTF8STRING *str = sk_ASN1_UTF8STRING_value(ss, i);
+        ASN1_UTF8STRING *dup = ASN1_STRING_dup(str);
 
-        if (!sk_ASN1_UTF8STRING_push(ctx->statusString, ASN1_STRING_dup(str)))
+        if (!dup || !sk_ASN1_UTF8STRING_push(ctx->statusString, dup)) {
```

Comment:
Please do not use pointer as boolean. I.E., use `dup == NULL`

Reply:
Changed.

> If you plan to submit more fixes, could you please consider signing a regular CLA?

I've brought this up with my supervisor